.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2 # use CircleCI 2.0
+
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
+    docker: # run the steps with Docker
+      - image: circleci/php:7.2-node-browsers # ...with this image as the primary container; this is where all `steps` will run
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: talentcloud
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: talentcloud
+          POSTGRES_DB: talentcloud
+    steps: # a set of executable commands
+      - checkout # special step to check out source code to working directory
+      - run: sudo apt update && sudo apt install zlib1g-dev postgresql-9.6 libpq-dev
```

Comment:
Curious why it's installing postgres inside a postgres:9.6 image?

Reply:
Redundant, just needs libpq-dev. Removed.